Signal/translations/bin/auto-genstrings

```diff
@@ -7,8 +7,10 @@ REPO_ROOT=$BIN_DIR/../../..
 cd $REPO_ROOT
 
 SSK_DIR="./SignalServiceKit/src"
+SAE_DIR="./SignalShareExtension"
+SM_DIR="./SignalMessaging"
 
-TARGETS="Signal/src ${SSK_DIR}"
+TARGETS="Signal/src ${SSK_DIR} ${SAE_DIR} ${SM_DIR}"
```

Comment:
Modified l10n string extraction script to scan `SAE` and `SignalMessaging`.  I've opened a card to modify the l10n scripts to copy l10n files from main app to `SAE`.

NOTE: I'm assuming it'll be easiest to have both the main app and the SAE use l10n files that contain the superset of strings in either.

Reply:
Question: should we follow the main app's convention of having a `src` folder?